### PR TITLE
fix: opens up close click to just top bar or sidebar/menu

### PR DESCRIPTION
**Pull request by jac-ob**

### CODE REVIEW

File: lib/composites/menu/events/main.js [hunk 1]
```diff
@@ -17,29 +17,33 @@ module.exports = (elements) => {
...
-    elements.trigger.addEventListener('click', onTriggerClick);
-  }
+      // hamburger clicks
+      elements.trigger.addEventListener('click', onTriggerClick);
+    }
+
+    if (topBar) {
+      const expandedDropdownItem = document.querySelector('[data-dropdown-expanded="true"]');
+
+      if (!topBar && expandedDropdownItem) {
```
Comment (schne324) [inline lib/composites/menu/events/main.js]: huh? we have:

```js
if (topBar) {
  if (!topBar && somethingThatWillNeverBeChecked) {}
}
```

Comment (schne324) [reply lib/composites/menu/events/main.js]: I think you may be confused by what you've named stuff here.  `topBar` represents an `<li class="data-dropdown-expanded">` within a `dqpl-top-bar`.  Probably makes sense to name it more descriptively

Comment (schne324) [reply lib/composites/menu/events/main.js]: try something like this:
```js
const hasTopBarDropdown = closest(target, '.dqpl-top-bar li[data-dropdown-expanded]');

if (hasTopBarDropdown) {
  const expandedDropdownItem = document.querySelector('[data-dropdown-expanded="true"]');

  if (expandedDropdownItem) {
    expandedDropdownItem.click(); // toggle the menu closed
  }
}
```

Comment (jac-ob) [reply lib/composites/menu/events/main.js]: was totally confused. This makes sense now. 👍

File: test/composites/menu/events/main.js [hunk 2]
```diff
@@ -67,14 +67,14 @@ describe('composites/menu/events/main', () => {
...
       const dropdown = elements.dropdown;
       assert.equal(dropdown.getAttribute('aria-expanded'), 'false');
-      fire(elements.topBar.querySelector('#drop-1'), 'click');
-      setTimeout(() => {
-        assert.equal(dropdown.getAttribute('aria-expanded'), 'true');
-        done();
-      }, 400); // give animation/other timeouts a change to do stuff
+      fire(elements.topBar.querySelector('.dd-trig'), 'click');
+      assert.equal(dropdown.getAttribute('aria-expanded'), 'true');
+      // setTimeout(() => {
```
Comment (schne324) [inline test/composites/menu/events/main.js]: `// commented out code!?!?!?`

**Pull request merged**